package/sbp_rtcm3_bridge/src/sbp_rtcm3_bridge.c

```diff
@@ -40,10 +40,20 @@ bool simulator_enabled_watch = false;
 
 pk_endpoint_t *rtcm3_pub = NULL;
 
+/* settings */
+
 static const char *const rtcm_out_modes[] = {"Legacy", "MSM4", "MSM5", NULL};
 enum { RTCM_OUT_MODE_LEGACY, RTCM_OUT_MODE_MSM4, RTCM_OUT_MODE_MSM5 };
 static u8 rtcm_out_mode = (u8)RTCM_OUT_MODE_MSM5;
 
+static float ant_height = 0.0;
+
+/* Antenna descriptor */
+static char ant_descriptor[RTCM_MAX_STRING_LEN] = "HXCGPS500";
```

Comment:
I'd make both of these settings as strings with the given default values - we may change the antenna to be a drop down list, but we should then support all IGS antenna types, which is hard to maintain, and we'd still want the ability to enter a custom string. To me this all seems to complex, so we should take the small hit on usability by pushing this onto the user.

Reply:
Agreed.